api/storage_service.cc

```diff
@@ -403,6 +403,19 @@ void set_storage_service(http_context& ctx, routes& r) {
         });
     });
 
+    ss::repair_async_status_until_finished.set(r, [&ctx](std::unique_ptr<request> req) {
+        return repair_get_status_until_finished(ctx.db, boost::lexical_cast<int>(req->get_query_param("id")))
```

Comment:
int? Not a larger type?

Reply:
It is historic which follows C*. We currently only return an int as the repair ID anyway.